packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -42,21 +57,34 @@ const Authorized = (props: Props) => {
       { extra: namesOfNonConfiguredPermissions }
     );
 
-  const isAuthorized = props.shouldMatchSomePermissions
+  const hasDemandedPermissions = props.shouldMatchSomePermissions
     ? hasSomePermissions(props.demandedPermissions, props.actualPermissions)
     : hasEveryPermissions(props.demandedPermissions, props.actualPermissions);
 
-  return <React.Fragment>{props.render(isAuthorized)}</React.Fragment>;
+  const hasDemandedActionRights = hasEveryActionRight(
+    props.demandedActionRights || [],
+    props.actualActionRights
+  );
+
+  return (
+    <React.Fragment>
+      {props.render(hasDemandedPermissions && hasDemandedActionRights)}
```

Comment:
Suggestion: we could maybe provide a second argument to the `render` prop that contains an object with each individual boolean, in case someone wants to know the exact condition results.

```js
props.render(
  hasDemandedPermissions && hasDemandedActionRights,
  { hasDemandedPermissions, hasDemandedActionRights }
)
```

This change is also backwards compatible. WDYT? Can this be useful for the UI?

Reply:
Nice idea but not needed so far no. I've just refactored the MC and not encountered a place where this is needed. I think it could be added later but given what we don't need now it feels a bit overly ambitous somehow.